spec/deaf_grandma_spec.rb

```diff
@@ -5,14 +5,16 @@
   let(:script) { DeafGrandma.new }
 
   it "says 'SPEAK UP SONNY!' when we speak regularly" do
-    expect(script.speak("Hi Grandma")).to eq "SPEAK UP SONNY!"
+    expect(script.speak("Hi Grandma")).to eq [true, "SPEAK UP SONNY!"]
```

Comment:
Thanks.  I agree that I felt that I was bending the rules a little, but I did not want to put an exit inside of speak.